src/main/kotlin/com/example/noswimmingserver/domain/reading_journal/presentation/ReadingJournalController.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.noswimmingserver.domain.reading_journal.presentation
+
+import com.example.noswimmingserver.domain.reading_journal.presentation.dto.request.CreateReadingJournalRequest
+import com.example.noswimmingserver.domain.reading_journal.service.CreateReadingJournalService
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
+import javax.validation.Valid
+
+@RequestMapping("/journal")
+@RestController
+class ReadingJournalController(
+    private val createReadingJournalService: CreateReadingJournalService,
+) {
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping("{book-id}")
+    fun createReadingJournal(
+        @PathVariable("book-id") bookId: Long,
+        @RequestBody @Valid request: CreateReadingJournalRequest
```

Comment:
🚫 **[ktlint]** <annotation> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Multiple annotations should not be placed on the same line as the annotated construct